Doc/c-api/bytes.rst

```diff
@@ -189,6 +189,19 @@ called with a non-bytes parameter.
    to *newpart* (i.e. decrements its reference count).
 
 
+.. c:function:: PyObject* PyBytes_Join(PyObject *sep, PyObject *iterable)
+
+   Similar to ``sep.join(iterable)`` in Python.
+
+   *sep* must be Python :class:`bytes` object.
+   *iterable* must be an iterable object.
```

Comment:
Please add that *iterable* should yiled objects that implement ```the :ref:`buffer protocol <bufferobjects>` ```.

It would also be nice to harmonize the wording for `PyBytes_Join()` and `PyUnicode_Join()`.

Explicitly specify the difference between `PyBytes_Join()` and `PyUnicode_Join()` (for sep == NULL). Or make `PyBytes_Join()` more similar to `PyUnicode_Join()`.

Reply:
I modified PyBytes_Join() to accept NULL separator.